core/isolation/wordpress-adapter.php

```diff
@@ -50,4 +50,21 @@ public function update_option( $option_key, $option_value ) : void {
 	public function add_option( $option_key, $option_value ) : void {
 		add_option( $option_key, $option_value );
 	}
+
+	public function get_referer(): string {
+		return wp_get_referer();
+	}
+
+	public function add_query_arg( $args, $url ) : string {
+		return add_query_arg( $args, $url );
+	}
+
+	public function has_custom_logo() : bool {
+		return has_custom_logo();
```

Comment:
Where is `has_custom_logo()` defined?

Reply:
is a wordpress method ( https://developer.wordpress.org/reference/functions/has_custom_logo/)

Reply:
I see. Sorry I missed it :)